visualizer/actionButton.js

```diff
@@ -279,4 +366,17 @@
     document.querySelector('#operations').innerHTML = '';
     document.querySelector('#attributes').innerHTML = '';
   });
+
+  // Hide the action context menu when Esc or Enter is pressed, any click happens, or another
+  // context menu is brought up.
+  document.addEventListener('click', hideContextMenu);
```

Comment:
Good idea. I think it's fine to merge this as-is, and clean up the duplication in a separate PR.

How about we move the contextMenu stuff to domUtil?